package.json

```diff
@@ -1,21 +1,22 @@
 {
-  "name":         "slugid",
-  "version":      "1.1.0",
-  "author":       "Jonas Finnemann Jensen <jopsen@gmail.com>",
-  "description":  "URL-safe base64 UUID encoder for generating 22 character slugs",
-  "license":      "MIT",
+  "name": "slugid",
+  "version": "2.0.0",
```

Comment:
Why the major version bump? I don't see anything here that breaks compatibility.

Let's not put work on consumers, if it's not needed.
I suspect a lot of down-streamers will have `"slugid": "^1.1.0"`, thus, next time they blow away their `package-lock.json` they'll be adopting `1.2.0`, which shouldn't give them any problems.

---
I could be missing something by I suspect this is mostly motivated by a desire to use the latest `uuid` package.
IMO there is nothing wrong with that :)
If nothing else because other packages that also require the latest `uuid` package will get the same package, instead of npm installing two versions of the package.

Anyways, I don't see any changes justifying a major bump in the [change log for `uuid`](https://github.com/kelektiv/node-uuid/blob/master/CHANGELOG.md) either.

Reply:
It's major because it drops support for older node versions.  I don't think `yarn` will catch that automatically and thus prevent upgrading to 1.2.0 on node 4 -- it will fail instead.

Reply:
Okay, that makes sense...